irods_builder.Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:22.04
+
+SHELL ["/bin/bash", "-c"]
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Re-enable apt caching for RUN --mount
+RUN rm -f /etc/apt/apt.conf.d/docker-clean && \
+    echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache
+
+# Make sure we're starting with an up-to-date image
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update && \
+    apt-get upgrade -y && \
+    apt-get autoremove -y --purge && \
+    rm -rf /tmp/*
+# To mark all installed packages as manually installed:
+#apt-mark showauto | xargs -r apt-mark manual
```

Comment:
this is ... needed?   delete-able?

Reply:
That was taken from the dev environment. It can be removed, but I left it in as a slight reminder to revisit in case the development environment changes the lines.

Reply:
I'm going to leave in for now.

Can remove/update as we approach 1.0.

Reply:
certainly.   resolving.